### PR TITLE
Converted measurements to a single pattern. Added function app metrics to dashboard.

### LINKED ISSUE #846: [Bug] Converted service event measurements to a single pattern
All service measurements should include three main events by following the pattern:

 - Started event: This event is emitted once an operation has been started
 - Completed event: This event is emitted once an operation is completed, regardless of the operation success/failure state
 - Failed event: This event is emitted when an operation completes with a failure state

### PR DESCRIPTION
**Pull request by lamaks**
#### Description of changes

Converted measurements to a single pattern.
Added function app metrics to dashboard.

screencapture-portal-azure-2020-05-28-12_02_10

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #846, #847
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/web-api-scan-runner/src/runner/runner.ts [hunk 1]
```diff
@@ -65,30 +68,37 @@ export class Runner {
 
         await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
 
+        let scanSucceeded: boolean;
         try {
             await this.webDriverTask.launch();
-            await this.scan(pageScanResult, scanMetadata.url);
-            this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
```
Comment (AhmedAbdoOrtiga) [inline packages/web-api-scan-runner/src/runner/runner.ts:71]: I'm wondering if "completedScanRequests" value is always 1, what value does it add if we already know that it is a "ScanRequestCompleted" event?

File: packages/web-api-scan-runner/src/runner/runner.ts [hunk 2]
```diff
@@ -65,30 +68,37 @@ export class Runner {
...
-            this.logger.trackEvent('ScanTaskFailed');
         } finally {
             const scanCompletedTimestamp: number = Date.now();
             const telemetryMeasurements: ScanTaskCompletedMeasurements = {
                 scanExecutionTime: (scanCompletedTimestamp - scanStartedTimestamp) / 1000,
                 scanTotalTime: (scanCompletedTimestamp - scanSubmittedTimestamp) / 1000,
+                completedScanTasks: 1,
             };
             this.logger.trackEvent('ScanTaskCompleted', undefined, telemetryMeasurements);
+            this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
```
Comment (AhmedAbdoOrtiga) [inline packages/web-api-scan-runner/src/runner/runner.ts:88]: I'm wondering if "completedScanRequests" value is always 1, what value does it add if we already know that it is a "ScanRequestCompleted" event?

File: packages/logger/README.md [hunk 3]
```diff
@@ -10,32 +10,36 @@ Licensed under the MIT License.
 All event names are defined in src/logger-event.ts
 
 -   **HealthCheck:** Sent each time check-health-func runs
--   **BatchPoolStats:** Sent by batch job manager. Custom measurements:
+-   **BatchPoolStats:** Sent by Batch job manager. Custom measurements:
     -   runningTasks: number of running tasks
     -   samplingIntervalInSeconds: the time interval to take pool metrics measurement
     -   maxParallelTasks: the pool configured maximum number of parallel tasks to run
--   **ScanRequestReceived:** Sent when requests received by post-scans-func. Custom measurements:
+-   **ScanRequestReceived:** Sent when request received by post-scans-func. Custom measurements:
```
Comment (AhmedAbdoOrtiga) [inline packages/logger/README.md:17]: NIT: Sent when a request is received by post-scans-func. Custom measurements:

File: packages/logger/README.md [hunk 4]
```diff
@@ -10,32 +10,36 @@ Licensed under the MIT License.
 All event names are defined in src/logger-event.ts
 
 -   **HealthCheck:** Sent each time check-health-func runs
--   **BatchPoolStats:** Sent by batch job manager. Custom measurements:
+-   **BatchPoolStats:** Sent by Batch job manager. Custom measurements:
```
Comment (AhmedAbdoOrtiga) [inline packages/logger/README.md:13]: NIT: why "Batch" not "batch"?

File: packages/logger/README.md [hunk 5]
```diff
@@ -10,32 +10,36 @@ Licensed under the MIT License.
...
--   **ScanRequestReceived:** Sent when requests received by post-scans-func. Custom measurements:
+-   **ScanRequestReceived:** Sent when request received by post-scans-func. Custom measurements:
     -   totalScanRequests
     -   pendingScanRequests
     -   rejectedScanRequests
--   **ScanRequestsAccepted:** Sent by web-worker each time a batch request is processed and split into multiple separate url scan requests. Custom measurements:
+-   **ScanRequestAccepted:** Sent by web-worker each time received request is processed and split into multiple separate scan requests. Custom measurements:
     -   acceptedScanRequests: number of scans that were accepted for the processing
--   **ScanRequestQueued:** Sent when scan requests are queued for processing. Custom measurements:
+-   **ScanRequestQueued:** Sent when scan request is queued for processing by Batch scan task. Custom measurements:
```
Comment (AhmedAbdoOrtiga) [inline packages/logger/README.md:23]: NIT: Sent when "a" scan request is queued for processing by Batch scan task. Custom measurements:

File: packages/logger/README.md (hunk 4 above)
Comment (lamaks) [reply packages/logger/README.md:13]: Batch as service name like Azure

**Pull request merged**